src/ontology/cl-edit.owl

```diff
@@ -30820,6 +30834,119 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32555229") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4033015 "retinal astrocyte")
 EquivalentClasses(obo:CL_4033015 ObjectIntersectionOf(obo:CL_0000127 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000966)))
 
+# Class: obo:CL_4033016 (secondary crest myofibroblast)
+
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033016 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033016 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033016 "2023-03-16T09:17:47Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31331942") oboInOwl:hasExactSynonym obo:CL_4033016 "secondary-crest myofibroblast")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33624948") oboInOwl:hasRelatedSynonym obo:CL_4033016 "SCMF")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25448080") oboInOwl:hasRelatedSynonym obo:CL_4033016 "SCMFs")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31331942") oboInOwl:hasRelatedSynonym obo:CL_4033016 "alveolar myofibroblast")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31331942") Annotation(oboInOwl:hasDbXref "PMID:33624948") rdfs:comment obo:CL_4033016 "A myofibroblast that is part of an alveoli during alveolarization. The contractile force of this cell elongates the secondary crest, while producing a framework of elastin and tenascin. During the maturation of the septa, secondary crest myofibroblasts, together with matrix fibroblasts, secretes metalloproteinases and other ECM‐remodeling proteins to thin the septal tip ECM. The secondary crest myofibroblast continues producing elastin, eventually undergoing apoptosis during adulthood.")
```

Comment:
The first sentence reads like a definition and there is otherwise currently no text definition annotated on this class. Unsure if this is intended.